orderBook.js

```diff
@@ -1,5 +1,45 @@
-function reconcileOrder(existingBook, incomingOrder) {
-  throw new Error('Not Implemented Yet')
+/* eslint-disable indent */
+const reconcileOrder = (existingBook, incomingOrder) => {
+
+    if (existingBook.length === 0) {
+        existingBook.push(incomingOrder)
+
+    } else {
```

Comment:
Since you have the else at the bottom you could skip this step. If existingBook is empty, when you call find on line 9 it will just return undefined without looping and then you'll skip to your else and add the order to the book on line 39.